sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -333,23 +367,32 @@ case class AlterTableExchangePartition(
  * unless 'purge' is true, but the metadata is completely lost.
  * An error message will be issued if the partition does not exist, unless 'ifExists' is true.
  * Note: purge is always false when the target is a view.
+ *
+ * The syntax of this command is:
+ * {{{
+ *   ALTER TABLE table DROP [IF EXISTS] PARTITION spec1[, PARTITION spec2, ...] [PURGE];
+ *   ALTER VIEW view DROP [IF EXISTS] PARTITION spec1[, PARTITION spec2, ...];
```

Comment:
I think we do not support `ALTER VIEW view DROP [IF EXISTS] PARTITION spec1[, PARTITION spec2, ...];` because the semantic of having partitions defined with a view is not defined?